.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+
+# Development
+
+We welcome contributions from the community!
+
+SimpleITK uses the [Git](http://git-scm.com/) distributed version control system. The main repository is hosted on [itk.org](https://itk.org/SimpleITK.git) and mirrored to [Github](https://github.com/SimpleITK/SimpleITK.git). We are currently using Github for pull requests, and issue tracking.
+
+# Contribute
+
+To submit code to SimpleITK, simply follow the standard GitHub workflow to make a pull request. More information can be found [here](https://guides.github.com/activities/forking/) and [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) regarding the workflow and getting your computer setup.
+
```

Comment:
May be worth indicating against which branch topic should be based of.  Should it always be `master` even when doing a fix for a release ?